config.json

```diff
@@ -685,7 +691,7 @@
         "name": "Bank Account",
         "uuid": "8beedf70-28b4-4cfb-ab53-309ee6f6aa78",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["classes", "conditionals", "numbers"],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/bank-account/solutions/TomPradat

This is pretty straightforward IMO